content/en/profiler/enabling/ddprof.md

```diff
@@ -17,7 +17,7 @@ further_reading:
 ---
 
 <div class="alert alert-warning">
-The native profiler for compiled languages, <code>ddprof</code> is in public beta. Datadog recommends evaluating the profiler in a non-sensitive environment before deploying in production.
+<code>ddprof</code> is in public beta. Datadog recommends evaluating the profiler in a non-sensitive environment before deploying in production.
```

Comment:
I thought this was too redundant with the text bellow

Reply:
```suggestion
<code>ddprof</code> is in beta. Datadog recommends evaluating the profiler in a non-sensitive environment before deploying in production.
```